sidekick/CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # Changelog
 
+## 0.7.2
+
+- Add documentation
+- Add example folder
```

Comment:
I'm missing the example

Reply:
A readme in the `example` folder is enough, like [melos](https://pub.dev/packages/melos/example) for example.